test/pages/api/ApiBrowserActionTest.php

```diff
@@ -13,5 +13,17 @@ public function testApiBrowser()
       ->ok()
       ->bodyContains('#swagger-ui');
   }
+
+  public function testApiBrowser_setup()
```

Comment:
didn't know that this works. very good

Reply:
unfortunately it doesn't so I switched back to plain file assertions :wink: